tests/integration_test/integration_test.sh

```diff
@@ -149,18 +149,26 @@ PIPELINE_TOOLS_DIR=${10}
 TENX_MODE=${11}
 TENX_VERSION=${12}
 TENX_DIR=${13}
-SS2_MODE=${14}
-SS2_VERSION=${15}
-SS2_DIR=${16}
-TENX_SUBSCRIPTION_ID=${17}
-SS2_SUBSCRIPTION_ID=${18:-"placeholder_ss2_subscription_id"}
-VAULT_TOKEN_PATH=${19}
-SUBMIT_WDL_DIR=${20}
-USE_CAAS=${21}
-USE_HMAC=${22}
-SUBMIT_AND_HOLD=${23}
-REMOVE_TEMP_DIR=${24:-"true"}
-COLLECTION_NAME=${25:-"lira-${LIRA_ENVIRONMENT}"}
+OPTIMUS_MODE=${14}
+OPTIMUS_VERSION=${15}
+OPTIMUS_DIR=${16}
+SS2_MODE=${17}
+SS2_VERSION=${18}
+SS2_DIR=${19}
+TENX_SUBSCRIPTION_ID=${20:-"placeholder_10x_subscription_id"}
+SS2_SUBSCRIPTION_ID=${21:-"placeholder_ss2_subscription_id"}
+OPTIMUS_SUBSCRIPTION_ID=${22:-"placeholder_optimus_subscription_id"}
```

Comment:
I think it's confusing that the pipeline variables are listed in an order (tenx, optimus, ss2) that is different from the order of the subscription ids (tenx, ss2, optimus). Could you make that ordering consistent? 

Alternatively, the variables could be reorganized so that the subscription ids are grouped with their corresponding pipelines:
```
TENX_MODE=${11}
TENX_VERSION=${12}
TENX_DIR=${13}
TENX_SUBSCRIPTION_ID=${14}
```

Reply:
Fixed as suggested! :octocat: